### PR TITLE
feat(ci): Enable double push 

### PR DESCRIPTION
**Pull request by tmdzk**
fix(ci): Enable double push 

## Summary

feat(ci): Enable double push 

## Test Plan

https://github.com/magma/magma/actions/runs/3527231739

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: .github/workflows/build_all.yml [hunk 1]
```diff
@@ -41,9 +44,9 @@ jobs:
       # Or is branch name when on release branch
       - name: Set Helm chart version
         run: |
-          if [ "${GITHUB_REF##*/}" = "master" ] ;then
+          if [ "${GITHUB_REF}" = "refs/heads/main" ] ;then
```
Comment (sebathomas) [inline .github/workflows/build_all.yml]: I don't think magma has a "main" branch?

File: .github/workflows/build_all.yml [hunk 2]
```diff
@@ -41,9 +44,9 @@ jobs:
...
             echo "VERSION=${GITHUB_RUN_NUMBER}" >> $GITHUB_ENV
-          elif [ "${EVENT_NAME}" = "pull_request" ]; then
+          elif [ "${GITHUB_EVENT_NAME}" = "pull_request" ]; then
```
Comment (sebathomas) [inline .github/workflows/build_all.yml:49]: What's the difference between event name and github event name?

File: .github/workflows/build_all.yml [hunk 3]
```diff
@@ -56,6 +59,16 @@ jobs:
           else
             orc8r/tools/helm/package.sh --deployment-type all
           fi
+      - name: Launch build and publish script - LF
+        run: |
+          if [ "${GITHUB_REF##*/}" = "master" ] ;then
+            orc8r/tools/helm/package.sh --deployment-type all --version $VERSION  --only-package
+          elif [ "${EVENT_NAME}" = "pull_request" ] ;then
```
Comment (sebathomas) [inline .github/workflows/build_all.yml]: This still uses EVENT_NAME, not GITHUB...?

File: .github/workflows/build_all.yml [hunk 4]
```diff
@@ -186,14 +201,25 @@ jobs:
...
           done
+
+          #Uploading it to the Linux foundation Registry
+          jf rt upload \
+            --recursive=false \
+            --detailed-summary \
+            --url https://linuxfoundation.jfrog.io/artifactory/ \
+            --user ${{ secrets.LF_JFROG_USERNAME }} \
+            --password ${{ secrets.LF_JFROG_PASSWORD }} \
+            ${{ env.IS_DRY }} \
```
Comment (sebathomas) [inline .github/workflows/build_all.yml]: Can this be set somewhere? If not, we should remove it.

File: .github/workflows/build_all.yml (hunk 1 above)
Comment (tmdzk) [reply .github/workflows/build_all.yml]: you're right

File: .github/workflows/build_all.yml (hunk 2 above)
Comment (tmdzk) [reply .github/workflows/build_all.yml:49]: I don't remember but in my other open PR for double push @Neudrino explained why we have to change it.

**Review by sebathomas — APPROVED** [review]

**Review by nstng — APPROVED** [review]

**Comment by sebathomas** [comment]
I think this needs a rebase for the CI to run.

**Comment by nstng** [comment]
@tmdzk this is 28 commits behind master - can we do a rebase?

**Comment by Neudrino** [comment]
Why was this separate PR opened and not the existing one #14345 improved?

The following PRs are related:
- https://github.com/magma/magma/pull/14345
- https://github.com/magma/magma/pull/14484
- https://github.com/magma/magma/pull/14527
- https://github.com/magma/magma/pull/14530
- https://github.com/magma/magma/pull/14532
- https://github.com/magma/magma/pull/14533

**Pull request merged**